.github/README.md

```
@@ -61,4 +334,4 @@
 
 ## Contact
 
-it@M - opensource@muenchen.de
+it@M - opensource@muenchen.de
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)